lib/ecto_mnesia/storage/migrator.ex

```diff
@@ -168,7 +170,9 @@ defmodule EctoMnesia.Storage.Migrator do
   # Helpers
   defp do_create_table(repo, table, type, attributes) do
     config = conf(repo)
+    attributes = if length(attributes) == 1, do: attributes ++ [:id], else: attributes
     tab_def = [{:attributes, attributes}, {config[:storage_type], [config[:host]]}, {:type, get_engine(type)}]
+    table = if String.valid?(table), do: String.to_atom(table), else: table
```

Comment:
Ecto tests are sending table names as strings, we must convert to atom to use on mnesia

Reply:
Good catch. It one of the changes in Ecto occurred lately.